src/main/java/ch/njol/skript/expressions/ExprArmorSlot.java

```diff
@@ -30,39 +31,52 @@
 import ch.njol.skript.util.slot.EquipmentSlot.EquipSlot;
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.Kleenean;
-import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.*;
 import org.bukkit.event.Event;
 import org.bukkit.inventory.EntityEquipment;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.Arrays;
-import java.util.Locale;
+import java.util.*;
 import java.util.stream.Stream;
 
 @Name("Armour Slot")
-@Description("Equipment of living entities, i.e. the boots, leggings, chestplate or helmet.")
+@Description({
+	"Equipment of living entities, i.e. the boots, leggings, chestplate or helmet.",
+	"`Body armor` is a special slot that can only be used for:",
+	"<ul>",
+	"<li>`Horses`: Horse armour (doesn't work on zombie or skeleton horses)</li>",
+	"<li>`Wolves`: Wolf Armor</li>",
+	"<li>`Llamas` (regular or trader): Carpet</li>",
+	"</ul>"
+})
 @Examples({
 	"set chestplate of the player to a diamond chestplate",
 	"helmet of player is neither a helmet nor air # player is wearing a block, e.g. from another plugin"
 })
 @Keywords("armor")
-@Since("1.0, 2.8.0 (Armour)")
+@Since("1.0, 2.8.0 (Armour), INSERT VERSION (body armour)")
```

Comment:
```suggestion
@Since("1.0, 2.8.0 (armor), INSERT VERSION (body armor)")
```

Reply:
I would appreciate it if you didnt make a change request on something that you had previously requested to be changed to that. It's quite confusing

Reply:
the u doesn't matter but the previous request asked for the A to become an a, which was not done.

Reply:
what? it's for standardizing the spelling, initially you didn't have a `body armor` section at all